docs/DafnyRef/README.md

```diff
@@ -65,11 +65,9 @@ definition file.
 Github uses rouge, via Jekyll. The syntax definition is a ruby-based file
 maintained in the rouge github repo.
 To modify the definition, follow the directions in
-[https://rouge-ruby.github.io/docs/file.LexerDevelopment.html]
-(https://rouge-ruby.github.io/docs/file.LexerDevelopment.html)
+https://rouge-ruby.github.io/docs/file.LexerDevelopment.html
 after setting up a development environment according to
-[https://rouge-ruby.github.io/docs/file.DevEnvironment.html]
-(https://rouge-ruby.github.io/docs/file.DevEnvironment.html).
+https://rouge-ruby.github.io/docs/file.DevEnvironment.html.
```

Comment:
I think the issue might have been the newline, but think we can improve on what was originally intended.
```
To modify the definition, follow [these](https://rouge-ruby.github.io/docs/file.LexerDevelopment.html) directions after [setting up](https://rouge-ruby.github.io/docs/file.DevEnvironment.html) a development environment.
```

Reply:
The newline (or lack of it) is an issue? Ok made changes as suggested above 884095a988e72105aefeb236d4e6343d7fd59e0c.

Reply:
That’s just my best guess as to why the Markdown processor didn't pick up the link.

Reply:
Oh it was those newlines that you were referring to ... yeah I think there cannot be any whitespace between the 2 components of the link i.e. between `[...]` and `(...)`.